lib/logidze/versioned_association.rb

```diff
@@ -0,0 +1,52 @@
+# frozen_string_literal: true
+module Logidze
+  module VersionedAssociation
+    def load_target
+      target = super
+
+      return target if inversed
+
+      time = owner.logidze_requested_ts
+
+      if target.is_a? Array
+        target.map! do |object|
+          object unless object.class.has_logidze?
```

Comment:
Not actually, we don't check, whether target class has logidze or not in the `#association` method :) And now I think, that it would be better to check it there really

Should we also add specs for the case of associations without logidze enabled?